backend/src/services/api-service/optimal/optimal-service.test.ts

```diff
@@ -53,6 +53,36 @@ describe('findOptimalSetsForMeal', () => {
   });
 });
 
+describe('findOptimalMonsForIngredient', () => {
+  it('shall find all optimal mons for an ingredient', () => {
```

Comment:
Yeah I think this works for now, looks like Slowbro and Slowking gather 1.5 tails each here so should be enough margin for now. I guess going forward we would probably want to just mock the call to set-cover class here and see that it is called with the correct parameters, since the functionality of the set-cover itself is outside this function's responsibility. Then assume the set cover handles its own testing (it does).

But yeah this is also how the other tests here work, it's fine for now

Reply:
After adding Ingredient Magnet mons to the results, I've filtered this list to only Pokemon with at least 1 full ingredient for the test assertions.